maxmin/max3/max3.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("The program finds the maximum of three numbers")
+	fmt.Println("Enter three numbers:")
+	var a, b, c float64
+	fmt.Scan(&a, &b, &c)
+	if a > b && a > c {
```

Comment:
Please implement max3 and min3 w/o using `if` statements. Think of how you could pipeline max (or min) calculations.